test/integration/evictions/evictions_test.go

```diff
@@ -290,18 +289,6 @@ func TestEvictionVersions(t *testing.T) {
 	}
 
 	allowedEvictions := []runtime.Object{
-		// v1beta1, no apiVersion/kind
-		&policyv1beta1.Eviction{
-			TypeMeta:      metav1.TypeMeta{},
-			ObjectMeta:    metav1.ObjectMeta{Name: pod.Name},
-			DeleteOptions: &metav1.DeleteOptions{DryRun: []string{metav1.DryRunAll}},
-		},
-		// v1beta1, apiVersion/kind
-		&policyv1beta1.Eviction{
-			TypeMeta:      metav1.TypeMeta{APIVersion: "policy/v1beta1", Kind: "Eviction"},
-			ObjectMeta:    metav1.ObjectMeta{Name: pod.Name},
-			DeleteOptions: &metav1.DeleteOptions{DryRun: []string{metav1.DryRunAll}},
-		},
```

Comment:
restore these tests... these are ensuring we remain compatible with both versions passed to the eviction subresource